crates/gpui/src/platform/windows/events.rs

```diff
@@ -828,7 +835,10 @@ fn handle_hit_test_msg(
         y: lparam.signed_hiword().into(),
     };
     unsafe { ScreenToClient(handle, &mut cursor_point).ok().log_err() };
-    if cursor_point.y > 0 && cursor_point.y < frame_y + padding {
+    if !state_ptr.state.borrow().is_maximized()
+        && cursor_point.y > 0
+        && cursor_point.y < frame_y + padding
+    {
```

Comment:
I could be wrong, but wouldn't this break the hover states of the close, max, and min buttons when you're maximized?

Reply:
No, It prevents it from returning `HTTOP` when maximized which is "the upper-horizontal border of a window." this controls whether the resize functionality at the top of the window is enabled, which there should not be any resize (cursor/functionality) because it is maximized.

[Source](https://learn.microsoft.com/en-us/windows/win32/inputdev/wm-nchittest)

Reply:
ah you're right, mb